tools/dotnet-linker/BackingFieldDelayHandler.cs

```diff
@@ -70,8 +70,10 @@ public static void ReapplyDisposedFields (DerivedLinkContext context, string ope
 				foreach (var ins in body.Instructions) {
 					switch (ins.OpCode.OperandType) {
 					case OperandType.InlineField:
-						var field = (ins.Operand as FieldReference)?.Resolve ();
-						if (!context.Annotations.IsMarked (field)) {
+						var fr = ins.Operand as FieldReference;
+						var field = fr?.Resolve ();
+						var isMarked = field is not null && context.Annotations.IsMarked (field);
+						if (!isMarked) {
```

Comment:
```suggestion
						if (ns.Operand is FieldReference fr && !context.Annotations.IsMarked (fr.Resolve())) {
```
Just playing ⛳ :)

Reply:
That's pretty much the original (broken code), because `fr.Resolve()` returns null (and IsMarked throws ArgumentNullException).